coil-base/src/main/java/coil/request/DefaultRequestOptions.kt

```diff
@@ -32,6 +33,22 @@ class DefaultRequestOptions(
     val networkCachePolicy: CachePolicy = CachePolicy.ENABLED
 ) {
 
+    fun copy(
+        dispatcher: CoroutineDispatcher = this.dispatcher,
+        transition: Transition = this.transition,
+        precision: Precision = this.precision,
+        bitmapConfig: Bitmap.Config = this.bitmapConfig,
+        allowHardware: Boolean = this.allowHardware,
+        allowRgb565: Boolean = this.allowRgb565,
+        placeholder: Drawable? = this.placeholder,
+        error: Drawable? = this.error,
+        fallback: Drawable? = this.fallback,
+        memoryCachePolicy: CachePolicy = this.memoryCachePolicy,
+        diskCachePolicy: CachePolicy = this.diskCachePolicy,
+        networkCachePolicy: CachePolicy = this.networkCachePolicy
+    ) = DefaultRequestOptions(dispatcher, transition, precision, bitmapConfig, allowHardware, allowRgb565, placeholder,
+        error, fallback, memoryCachePolicy, diskCachePolicy, networkCachePolicy)
+
```

Comment:
Can't make this a data class I guess?

Reply:
Technically this could be a `data` class, however I'm starting to avoid `data` for public api classes due to the reasons [here](https://jakewharton.com/public-api-challenges-in-kotlin/). Mostly want to avoid creating component functions so I can still control property ordering in the constructor in future versions.

Reply:
Ah good call 👍